src/server/api.js

```diff
@@ -63,16 +63,42 @@ function submitApplication(req, res) {
       if (err) {
         debug(`voaService response had error ${err}`);
         // TODO(awong): This may leak server config info on error. Is that a problem?
-        res.status(500).send({ error: err });
+        res.status(500).json({ error: err });
```

Comment:
Can you ping Robbie to ensure .json is right versus .send?  I *think* it's better to coerce to json in the event we make a programming error, but not sure.